vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
Yeah, theoretically there should be only nil or string going out of the API. But property.try(:to_i) || 0 will work in 100% of the cases. So not sure whether to remove it. :-)